### ISSUE TITLE
Deep explaination regards

### ISSUE THREAD

**Issue by karthika-devi** [suriyadeepan/tf_seq2seq_wrapper#65]
Hi sir,
      As I am new to seq2seq model.I understand how you construct data and why you are using rnn with lstm.But I want to know whether seq2seq can be used for conversation. please explain the functionality of this   tf.contrib.legacy_seq2seq.embedding_rnn_seq2seq,tf.contrib.rnn.MultiRNNCell,tf.contrib.rnn.BasicLSTMCell,tf.contrib.rnn.DropoutWrapper

I can understand theoritically what is multiRNN,lstm,dropout,seq2seq but how input get interacted with all those things and why we construcrt this is needed for me to proceed my work

After training get finished I got same word repeated as a result eg:
Question: [big unk so far how about you]; 
answer : [classic classic classic classic classic classic classic classic classic classic classic classic classic classic classic classic classic tie tie tie tie wendell wendell wendell wendell] please guide me  why I got this answer and where I want to improve 

Thank you

**Comment by karanpande** [comment]
Each of them functions you mentioned works on the input in a different manners, for example: dropout will prevent the model from overfitting (it just drops nodes). LSTM will help the model not to make grammatical mistakes. Rnn and lstm work in tandem. And so on

With regards to you output-
There could be multiple reasons why your output is the way it is.
1: May be the vocabulary size is small try to increase it
2: try training the model on a different dataset
3: increase the number of epochs



Sent from my iPhone

On Jun 24, 2018, at 2:05 PM, karthika-devi <notifications@github.com<mailto:notifications@github.com>> wrote:


Hi sir,
As I am new to seq2seq model.I understand how you construct data and why you are using rnn with lstm.But I want to know whether seq2seq can be used for conversation. please explain the functionality of this tf.contrib.legacy_seq2seq.embedding_rnn_seq2seq,tf.contrib.rnn.MultiRNNCell,tf.contrib.rnn.BasicLSTMCell,tf.contrib.rnn.DropoutWrapper

I can understand theoritically what is multiRNN,lstm,dropout,seq2seq but how input get interacted with all those things and why we construcrt this is needed for me to proceed my work

After training get finished I got same word repeated as a result eg:
Question: [big unk so far how about you];
answer : [classic classic classic classic classic classic classic classic classic classic classic classic classic classic classic classic classic tie tie tie tie wendell wendell wendell wendell] please guide me why I got this answer and where I want to improve

Thank you
